test/models/network.js

```diff
@@ -83,6 +83,29 @@ describe("Network", function () {
 			expect(network2.username).to.equal("InvalidNick");
 		});
 
+		it("realname should be set to nick only if realname is empty", function () {
+			Helper.config.defaults.nick = "";
```

Comment:
This is pretty mysterious to me. Might be worth a comment explaining why this is necessary (if it is). I'm hoping `Helper` knows how to reset its configuration between test runs too since we're doing this in other tests as well.

Reply:
Honestly? No clue, I just copy pasted from another test.

I guess the reason is to make it less likely to make the test pass by chance due to the defaults.
But I on purpose took stupid names to avoid that.

I'll remove it and see what happens